gossip/src/duplicate_shred_handler.rs

```diff
@@ -25,6 +26,7 @@ const MAX_NUM_ENTRIES_PER_PUBKEY: usize = 128;
 const BUFFER_CAPACITY: usize = 512 * MAX_NUM_ENTRIES_PER_PUBKEY;
 
 type BufferEntry = [Option<DuplicateShred>; MAX_NUM_CHUNKS];
+type DuplicateSlotSender = Sender<Slot>;
```

Comment:
please remove this typedef.
For someone reading the code, `Sender<Slot>` is more explicit what actual type we are working with, and does not need to jump elsewhere to find the definition.